user.conf

```diff
@@ -0,0 +1,2 @@
+search = on
```

Comment:
Do we need this file in master?

Reply:
This file is using only for integration-tests (by docker-compose file)